docs/cloud-native-security/kspm-faq.asciidoc

```diff
@@ -0,0 +1,51 @@
+[[kspm-faq]]
+== KSPM FAQ
+Frequently asked questions related to the Kubernetes Security Posture Management (KSPM) integration.
+
+[discrete]
+== General KSPM FAQ
+
+*Can I evaluate the security posture of my Amazon EKS clusters?*
+Yes. KSPM currently supports the security posture evaluation of Amazon EKS and unmanaged Kubernetes clusters.
+
+*How often is my cluster’s security posture evaluated?*
+Clusters are evaluated upon deployment of a KSPM integration, and every four hours after that.
+
+[discrete]
+=== Findings page FAQ
+
+*Are all the findings on the Findings page current?*
+Yes. Only the most recent findings appear on the Findings page.
+
+*Can I build custom visualizations and dashboards that incorporate findings data?*
+Yes. You can use {kib}'s custom visualization capabilities with findings data. Current findings data can be accessed using the `logs-cloud_security_posture.findings_latest-*` index pattern, while historical findings data can be accessed using the `logs-cloud_security_posture.findings-*` index pattern. To learn more, refer to {kibana-ref}/dashboard.html[Dashboards and visualizations].
```

Comment:
Do you need the second half about where to find current and historical data? Or is that information more relevant to the next question? 

```suggestion
Yes. You can use findings data to build {kibana-ref}/dashboard.html[custom visualizations] in {kib}. 
```